Tools/TestWebKitAPI/Tests/WebKitCocoa/WKWebExtensionAPILocalization.mm

```diff
@@ -107,9 +107,11 @@
 TEST(WKWebExtensionAPILocalization, i18n)
 {
     NSArray<NSString *> *preferredLocaleIdentifiers = NSLocale.preferredLanguages;
-    NSMutableOrderedSet<NSString *> *acceptedLanguages = [NSMutableOrderedSet orderedSetWithArray:preferredLocaleIdentifiers];
-    for (NSString *localeIdentifier in preferredLocaleIdentifiers)
+    NSMutableOrderedSet<NSString *> *acceptedLanguages = [NSMutableOrderedSet orderedSetWithCapacity:preferredLocaleIdentifiers.count];
+    for (NSString *localeIdentifier in preferredLocaleIdentifiers) {
+        [acceptedLanguages addObject:localeIdentifier];
         [acceptedLanguages addObject:[NSLocale localeWithLocaleIdentifier:localeIdentifier].languageCode];
+    }
```

Comment:
The localIdentifier and the language code need to both be added?

Reply:
Yeah Like "en-US", followed by "en". More specific to least specific. This matches Chrome and what we ship today.